docs/schema/constants/star-type.json

```diff
@@ -1,15 +1,16 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "type": "integer",
-    "title": "StarType",
-    "description": "",
+    "title": "Star Type",
+    "default": 1,
+    "description": "Star type, `1` for Star, `2` for Polygon",
     "oneOf": [
         {
-            "title": "Star",
+            "title": "Star Type Default",
```

Comment:
I don't think marking "Default" in the title is a good pattern

Reply:
can you please suggest a better name? because "Star Type" is taken in the parent's title above